hyperjumptech/monika
# Add cli documentations

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Closes #292 
2. Adding documentation shortuct to sidebar

## How did you implement / how did you fix it  
1.  Add shortcut to manifest

## How to test  
1. clone the project
2. move tot he docs/nextjs directory `cd docs`
3. type `npm ci` then  `npm run dev`
4. point your browser to the local server
5. you should see the updated sidebar

![image](https://user-images.githubusercontent.com/964106/127142836-1b08583e-11ff-4095-bdc7-025204e34c4e.png)
